packages/fiori-freestyle-writer/templates/listdetail/add/webapp/controller/List.controller.ts

```diff
@@ -164,13 +165,13 @@ export default class List extends BaseController {
      * are applied to the list, which can also mean that they
      * are removed from the list, in case they are
      * removed in the ViewSettingsDialog.
-     * @param event the confirm event
+     * @param oEvent the confirm event
      */
-    public onConfirmViewSettingsDialog(event: Event) {
+    public onConfirmViewSettingsDialog(oEvent: Event) {
         <%if (template.settings.entity.numberProperty) {%>
-        var aFilterItems = oEvent.getParameters().filterItems,
-            aFilters = [],
-            aCaptions = [];
+        const aFilterItems: any[] = (oEvent.getParameters() as any).filterItems;
+        let aFilters:  Filter[] = [];
+        const aCaptions: any[] = [];
```

Comment:
Should it be string[] type since `aCaptions.push(oItem.getText());`

Reply:
Done with https://github.com/SAP/open-ux-tools/pull/902

Reply:
Thanks